R/pc.R

```diff
@@ -0,0 +1,132 @@
+library(haven) #Load xpt
+library(dplyr) #apply distincts
+library(lubridate)
+library(ggplot2)
+setwd('D:/Users/ARodr181/Downloads/sdtm')

```

Comment:
@toni-1991 @RealSlimMahdi the code should take input data from the rda files under this repo in data folder. this way anyone could re-create the test data themselves using the code if they so wish